### ISSUE TITLE
[Update request]: Mention that hugo extended is necessary to host the website locally

### ISSUE THREAD

**Issue by StefKirsch** [tilburgsciencehub/website#747]
# Hugo extended necessary to host website locally

__Background__

Apparently Hugo extended is necessary to host the website locally. When trying to build the website with the regular version is used (in version 0.155.0), I am getting the following error:
`Error: error building site: TOCSS: failed to transform "scss/bootstrap.scss" (text/x-scss). Check your Hugo installation; you need the extended version to build SCSS/SASS with transpiler set to 'libsass'.: this feature is not available in your current Hugo version, see https://goo.gl/YMrWcn for more information`

__What needs to be updated?__

The tutorial [Contribute to Tilburg Science Hub](https://tilburgsciencehub.com/tutorials/more-tutorials/contribute-to-tilburg-science-hub/pullrequests/) explains the steps in detail on how to write content for the website and how to preview it with hugo, However, it does not mention to download the extended version of hugo.

I think it's best to mention it right here in the [tutorial](https://tilburgsciencehub.com/tutorials/more-tutorials/contribute-to-tilburg-science-hub/pullrequests/).

**Comment by thierrylahaije** [comment]
@hannesdatta - I looked into it, but there is nothing much we can do about this except as @StefKirsch said add this to the Contribute to TSH page and the specific tutorial. To clarify, the basic version of Hugo does not support SCSS compilation using LibSass or Dart Sass. The problem is that our Hugo website uses SCSS (Sass) files... Since we cannot eliminate these SCSS files (TSH uses a customized Bootstrap Framework for responsiveness and styling), we can only solve this by updating our versions to the extended version. 

I could not find out whether this changed recently in the updates of Hugo or this was already the case for a longer time, since I already used the extended version since using Hugo.

Do you want me to update the specific pages Stefan mentioned?

**Comment by hannesdatta** [comment]
Exactly, @thierrylahaije - just update the necessary pages so that potential contributors immediately get directed to Hugo extended, not the basic Hugo version. 

Thanks!

**Comment by thierrylahaije** [comment]
@hannesdatta I did some research on this issue and found the following on the website of Hugo:

<img width="572" alt="image" src="https://github.com/tilburgsciencehub/website/assets/65568251/2a901484-505f-4e4e-81a4-70a2274717c8">

I tried it myself, and the command mentioned indeed installs the extended version. Therefore, I do not think the pages need adjustment, since they already mention the correct command. 

@StefKirsch two questions to reproduce your issue:
- Was it a long time ago you installed Hugo or recently? 
- Did you use brew to install Hugo as mentioned in the get Hugo tutorial?

**Comment by StefKirsch** [comment]
Hi @hannesdatta,

Regarding your questions:
- I installed hugo in July this year. 
- I used chocolatery to install hugo, as is explained [here ](https://tilburgsciencehub.com/tutorials/code-like-a-pro/hugo-website/get-hugo/) (I'm on Windows).  I used the command `choco install hugo -confirm`. It looks like on the website, it now says to install hugo with `choco install hugo-extended -confirm` which implicitly recommends you should get the extended version. I'm not sure what it said back in July, but it seems pretty clear now that yoou should indeed get the extended version 😊
